graph/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  proxy:
+    build: .
+    environment:
+      RPC: "https://testnet.sapphire.oasis.dev"
+  graph-node:
+    image: graphprotocol/graph-node
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
+      - postgres
+      - proxy
+    ports:
+      - '127.0.0.1:8000:8000'
+      - '127.0.0.1:8001:8001'
+      - '127.0.0.1:8020:8020'
+      - '127.0.0.1:8030:8030'
+      - '127.0.0.1:8040:8040'
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    environment:
+      postgres_host: postgres
+      postgres_user: graph-node
+      postgres_pass: let-me-in
```

Comment:
Is this intended to be deployed? If not, could you please rename this directory to `example-graph` or something that evidences that it's not a good idea to deploy this directly? Either that or add a `README.md` about how to make it production-ready.

Reply:
I think it might be a good idea to move it out to foreign .env file